### PR TITLE
Enable for course responsible

### PR DESCRIPTION
**Pull request by me97esn**

### CODE REVIEW

File: backend/src/api/error.ts [hunk 1]
```diff
@@ -230,8 +230,12 @@ function isOperationalOrRecoverableError(err) {
 // Find a wrapped programmer error
 function getOrigProgrammerError(error) {
   const { err } = error;
+
   let result;
-  if (isOperationalOrRecoverableError(err)) {
+  if(! err){
+    // This is a leaf, with no wrapped errors. 
+    result = error 
+  } else if (isOperationalOrRecoverableError(err)) {
```
Comment (me97esn) [inline backend/src/api/error.ts:238]: When I tested this branch my admin token had  expired. This caused errors with no wrapped err property, which wasn't properly logged. By returning the original error if there is no wrapped err property, the error is properly logged

File: backend/package.json [hunk 2]
```diff
@@ -8,7 +8,6 @@
   "main": "index.js",
   "scripts": {
     "start": "ts-node --transpile-only src/app.ts",
-    "start-js": "node dist/app.js",
```
Comment (me97esn) [inline backend/package.json:11]: I see no other references to dist, I assume that this script is old and not used anymore

File: backend/src/api/error.ts (hunk 1 above)
Comment (jhsware) [reply backend/src/api/error.ts:238]: LGTM

**Review by jhsware — APPROVED** [review]
LGTM

NOTE: I have not double checked that you chose the correct value for "COURSE_RESPONSIBLE_ROLE", let me know if you want me to do that.

**Pull request merged**